_data/releases.json

```
@@ -1,86 +1,86 @@
 {
   "legacy": {
-    "version": "23.02",
-    "date": "Jan 12 2023"
-  },
-  "stable": {
     "version": "23.04",
     "date": "Apr 13 2023"
   },
-  "nightly": {
+  "stable": {
     "version": "23.06",
+    "date": "Jun 8 2023"
+  },
+  "nightly": {
+    "version": "23.08",
     "cudf_dev": {
-      "start": "Mar 23 2023",
-      "end": "May 17 2023",
+      "start": "May 18 2023",
+      "end": "Jul 19 2023",
       "days": 40
     },
     "other_dev": {
-      "start": "Mar 30 2023",
-      "end": "May 24 2023",
+      "start": "May 25 2023",
+      "end": "Jul 26 2023",
       "days": 40
     },
     "cudf_burndown": {
-      "start": "May 18 2023",
-      "end": "May 24 2023",
+      "start": "Jul 20 2023",
+      "end": "Jul 26 2023",
       "days": 5
     },
     "other_burndown": {
-      "start": "May 25 2023",
-      "end": "May 31 2023",
-      "days": 4
+      "start": "Jul 27 2023",
+      "end": "Aug 2 2023",
+      "days": 5
     },
     "cudf_codefreeze": {
-      "start": "May 25 2023",
-      "end": "June 6 2023",
+      "start": "Jul 27 2023",
+      "end": "Aug 8 2023",
       "days": 9
     },
     "other_codefreeze": {
-      "start": "Jun 1 2023",
-      "end": "Jun 6 2023",
+      "start": "Aug 3 2023",
+      "end": "Aug 8 2023",
       "days": 4
     },
     "release": {
-      "start": "Jun 7 2023",
-      "end": "Jun 8 2023",
+      "start": "Aug 9 2023",
+      "end": "Aug 10 2023",
       "days": 2
     }
   },
   "next_nightly": {
-    "version": "23.08",
+    "version": "23.10",
     "cudf_dev": {
-      "start": "May 18 2023",
-      "end": "Jul 19 2023",
-      "days": 40
+      "start": "Jul 20 2023",
+      "end": "Sep 20 2023",
+      "days": 42
     },
     "other_dev": {
-      "start": "May 25 2023",
-      "end": "Jul 26 2023",
-      "days": 40
+      "start": "Jul 27 2023",
+      "end": "Sep 20 2023",
+      "days": 42
     },
     "cudf_burndown": {
-      "start": "Jul 20 2023",
-      "end": "Jul 26 2023",
+      "start": "Sep 21 2023",
+      "end": "Sep 27 2023",
       "days": 5
     },
     "other_burndown": {
-      "start": "Jul 27 2023",
-      "end": "Aug 2 2023",
+      "start": "Sep 28 2023",
+      "end": "Oct 4 2023",
       "days": 5
     },
     "cudf_codefreeze": {
-      "start": "Jul 27 2023",
-      "end": "Aug 8 2023",
+      "start": "Sep 28 2023",
+      "end": "Oct 10 2023",
       "days": 9
     },
     "other_codefreeze": {
-      "start": "Aug 3 2023",
-      "end": "Aug 8 2023",
+      "start": "Oct 5 2023",
+      "end": "Oct 10 2023",
       "days": 4
     },
     "release": {
-      "start": "Aug 9 2023",
-      "end": "Aug 10 2023",
+      "start": "Oct 11 2023",
+      "end": "Oct 12 2023",
       "days": 2
     }
-  }
+  },
```

JSON doesn't permit trailing commas.
```suggestion
  }
```

---

Fixed. I'm surprised my local jekyll nor Netlify complained!